src/utils.py

```diff
@@ -102,10 +102,10 @@ def parse_splice(cdsEffect, position, strand, fasta):
             mylen = abs(int(sub[0]) - int(sub[1]))+1
         else:
             if sub[0] != '0' and sub[1] != '0':
-                if int(main[1]) - int(main[0]) == 1:
-                    mylen = seqlen
-                else:
-                    raise ValueError('ERROR: main position has to match for both intronic else one of them must be an exon: ')
+                #if int(main[1]) - int(main[0]) == 1:
```

Comment:
Why not just delete the commented out lines... they would be retained in git history for the file if we needed to revert to an earlier commit

Reply:
done https://github.com/lifeomic/foundation-xml-fhir/commit/c2972b91f4a391bbb367710887d8bba1b8d1c6c0